doc/tools/doc_status.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
I changed `#!/usr/bin/env python3` to `#!/usr/bin/env python` since the latter is used far more often within the codebase.